### PR TITLE
Deprecate old sandblast integrations

### PR DESCRIPTION
**Pull request by LiorQM**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by LiorQM** [comment]
@DanilVil @JudahSchwartz 
Hi, 
I deprecated the old SandBlast integrations and updated the name of new one as requested.
🙂

File: Packs/SandBlastAppliance/pack_metadata.json [hunk 1]
```diff
@@ -18,5 +18,6 @@
     "marketplaces": [
         "xsoar",
         "marketplacev2"
-    ]
+    ],
+    "hidden": true
```
Comment (yaakovi) [inline Packs/SandBlastAppliance/pack_metadata.json:22]: Do not hide the pack yet.
We are allowing customers to migrate before removing the pack from the marketplace.
```suggestion
    ]
```

File: Packs/Sandblast/pack_metadata.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
     "marketplaces": [
         "xsoar",
         "marketplacev2"
-    ]
+    ],
+    "hidden": true
```
Comment (yaakovi) [inline Packs/Sandblast/pack_metadata.json:22]: same here
```suggestion
    ]
```

**Review by yaakovi — APPROVED** [review]

**Comment by yaakovi** [comment]
@LiorQM please check the build failures

File: Packs/SandBlastAppliance/pack_metadata.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "Check Point Sandblast Appliance",
-    "description": "Query, upload and download data using Check Point Sandblast on local gateway.",
+    "name": "Check Point Sandblast Appliance (Deprecated)",
+    "description": "Deprecated. Use Check Point Threat Emulation (SandBlast) instead. Query, upload and download data using Check Point Sandblast on local gateway.",
```
Comment (bakatzir) [inline Packs/SandBlastAppliance/pack_metadata.json:3]:
```suggestion
    "description": "Deprecated. Use Check Point Threat Emulation (SandBlast) instead.",
```

File: Packs/Sandblast/pack_metadata.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "Check Point Sandblast Cloud Services",
-    "description": "Query, upload and download data using Check Point Sandblast on cloud.",
+    "name": "Check Point Sandblast Cloud Services (Deprecated)",
+    "description": "Deprecated. Use Check Point Threat Emulation (SandBlast) instead. Query, upload and download data using Check Point Sandblast on cloud.",
```
Comment (bakatzir) [inline Packs/Sandblast/pack_metadata.json:3]:
```suggestion
    "description": "Deprecated. Use Check Point Threat Emulation (SandBlast) instead.",
```

File: Packs/SandBlastAppliance/Integrations/CheckPointSandblastAppliance/CheckPointSandblastAppliance.yml [hunk 5]
```diff
@@ -1,10 +1,11 @@
+deprecated: true
 commonfields:
   id: Check Point Sandblast Appliance
   version: -1
 name: Check Point Sandblast Appliance
 display: Check Point Sandblast Appliance
```
Comment (yaakovi) [inline Packs/SandBlastAppliance/Integrations/CheckPointSandblastAppliance/CheckPointSandblastAppliance.yml:6]:
```suggestion
display: Check Point Sandblast Appliance (Deprecated)
```

File: Packs/Sandblast/Integrations/CheckPointSandblast/CheckPointSandblast.yml [hunk 6]
```diff
@@ -1,10 +1,11 @@
+deprecated: true
 commonfields:
   id: Check Point Sandblast
   version: -1
 name: Check Point Sandblast
 display: Check Point Sandblast Cloud Services
```
Comment (yaakovi) [inline Packs/Sandblast/Integrations/CheckPointSandblast/CheckPointSandblast.yml:6]:
```suggestion
display: Check Point Sandblast Cloud Services (Deprecated)
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**